backend/coins/btc/handlers/handlers.go

```diff
@@ -383,13 +383,26 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
 	balance, err := handlers.account.Balance()
 	if err != nil {
-		return nil, err
+		return struct {
+			Success      bool   `json:"success"`
+			ErrorMessage string `json:"ErrorMessage"`
```

Comment:
camelCase `errorMessage`, but shouldn't it be 

    ErrorCode    string `json:"errorCode,omitempty"``

instead if it's the syncing error code?

Also, if we are not making use of either `errorMessage` nor `errorCode` in the frontend (e.g. if the result is simply ignored in case of error), you could omit both and just send `success: false`.